src/rudderstack.ts

```diff
@@ -105,42 +105,34 @@ export class RudderStack {
   }
 
   init() {
-    const isProduction = process.env.CIRCLE_JOB === "release_production";
-    const isStaging = process.env.CIRCLE_JOB === "release_staging";
-
-    let RUDDERSTACK_KEY;
-    if (isProduction) {
-      RUDDERSTACK_KEY = process.env.RUDDERSTACK_PRODUCTION_KEY;
-    } else if (isStaging) {
-      RUDDERSTACK_KEY = process.env.RUDDERSTACK_STAGING_KEY;
-    }
+    const is_production = process.env.CIRCLE_JOB === "release_production";
 
+    const RUDDERSTACK_KEY = is_production
+        ? process.env.RUDDERSTACK_PRODUCTION_KEY
+        : process.env.RUDDERSTACK_STAGING_KEY;
```

Comment:
I've updated this condition so that users should now be able to test the Rudderstack events locally and not in staging environment, provided that they have set the environment variables for `process.env.RUDDERSTACK_STAGING_KEY` and `process.env.RUDDERSTACK_URL`

Reply:
i.e. removed `const isStaging = process.env.CIRCLE_JOB === "release_staging";` check